commit 3da3e74521d4 (HEAD)
Author: Thomas Nguyen <tnguyen@mozilla.com>
Date:   Fri Sep 29 01:46:00 2017 -0400

    Bug 1403473 - Init loggingEnable for hashcompleter. r=dimi, a=sledru
    
    MozReview-Commit-ID: 1BpErwmbgCM
    
    --HG--
    extra : amend_source : 5993fe27195003ac1fa77f89b4d14187cbff591e
---
 toolkit/components/url-classifier/nsUrlClassifierHashCompleter.js | 2 ++
 1 file changed, 2 insertions(+)

diff --git toolkit/components/url-classifier/nsUrlClassifierHashCompleter.js toolkit/components/url-classifier/nsUrlClassifierHashCompleter.js
index eac72dadabd3..16b1b7dc7c52 100644
--- toolkit/components/url-classifier/nsUrlClassifierHashCompleter.js
+++ toolkit/components/url-classifier/nsUrlClassifierHashCompleter.js
@@ -184,6 +184,8 @@ function HashCompleter() {
 
   Services.obs.addObserver(this, "quit-application");
   Services.prefs.addObserver(PREF_DEBUG_ENABLED, this);
+
+  loggingEnabled = Services.prefs.getBoolPref(PREF_DEBUG_ENABLED);
 }
 
 HashCompleter.prototype = {
